sycl/doc/extensions/proposed/sycl_ext_oneapi_free_function_kernels.asciidoc

```diff
@@ -323,6 +325,118 @@ Otherwise `value` is `false`.
 The helper trait `is_kernel_v` provides the value of `value`.
 |====
 
+=== New free functions to launch a kernel
+
+This extension adds the following helper which captures a kernel function
+address as a template parameter.
+
+[frame=all,grid=none,separator="@"]
+!====
+a@
+[source,c++]
+----
+namespace sycl::ext::oneapi::experimental {
+
+template<auto *Func>
```

Comment:
There is a note right after these changes stating:

```
[_Note:_ Many of the functions in this section have a template parameter
`Func`, which identifies a free function kernel.
This kernel function may be defined in any translation unit in the application.
_{endnote}_]
```
Would it make sense to move that note further up?